lib/real-device.js

```diff
@@ -314,14 +314,15 @@ export class RealDevice {
    * @param {import('./driver').XCUITestDriverOpts} opts
    * @returns {Promise<void>}
    */
-  async reset({bundleId, fullReset, platformVersion}) {
```

Comment:
should we also update other places where this method is called to adjust the count of arguments?

Reply:
The given arg is `this.opts`, so I think this removal is pretty safe. Just `platformVersion` from the given `opts` will not be picked up by this